source/code/test/private/game.cxx

```diff
@@ -381,6 +394,22 @@ void MyGame::on_update(ice::EngineFrame& frame, ice::EngineRunner& runner, ice::
 
     runner.graphics_frame().enqueue_pass("default"_sid, "game.render"_sid, _game_gfx_pass.get());
 
+    ice::shards::inspect_each<ice::c8utf const*>(
```

Comment:
This could be replaced with `ice::shards::transform_all`, but need double checking.

Reply:
This does not work, as prev_frame is immutable, and we cannot mutate shards.
Bu we might add a new api to move and transform shards from one array to another.